### PR TITLE
identify external repository

### PR DESCRIPTION
**Pull request by anara123**
## Status
- [ ] In Progress
- [x] Ready
- [ ] In Hold - (Reason for hold)

## Related Issues
fixes: link to the issue

## Description
Identify if demisto-sdk running inside external repository

## Screenshots
Paste here any images that will help the reviewer

## Must have
- [ ] Tests
- [ ] Documentation

### CODE REVIEW

File: demisto_sdk/commands/common/tools.py [hunk 1]
```diff
@@ -740,13 +741,14 @@ def get_common_server_dir_pwsh(env_dir):
     return _get_common_server_dir_general(env_dir, 'CommonServerPowerShell')
 
 
-def is_private_repository():
+def is_external_repository():
     """
     Returns True if script executed from private repository
```
Comment (reutshal) [inline demisto_sdk/commands/common/tools.py:746]:
```suggestion
    Returns True if script executed from an external repository
```

File: demisto_sdk/commands/create_artifacts/content_creator.py [hunk 2]
```diff
@@ -44,7 +44,7 @@ def __init__(self, artifacts_path: str, content_version='', content_bundle_path=
         self.artifacts_path = artifacts_path if artifacts_path else '/home/circleci/project/artifacts'
         self.content_version = content_version
         self.preserve_bundles = preserve_bundles
-        self.only_packs = tools.is_private_repository() or packs
+        self.only_packs = tools.is_external_repository() or packs
```
Comment (reutshal) [inline demisto_sdk/commands/create_artifacts/content_creator.py:47]: Not sure about the logic here. Before, `only packs` was true if it was a private repo and now it is true if it is an external one?

**Comment by anara123** [comment]
I just changed the names. External Repo is a new concept . Previously I called them private repo, but the more precise name is external repo.

File: demisto_sdk/commands/create_artifacts/content_creator.py (hunk 2 above)
Comment (anara123) [reply demisto_sdk/commands/create_artifacts/content_creator.py:47]: The logic is the same. Just renamed how we call these repos

**Pull request merged**